### PR TITLE
WIP: Add support for missing inline elements

### PR DESCRIPTION
**Pull request by giacomocavalieri**
> 🚧 This is still a WIP since there are still some tests that are failing and I'm currently ignoring

This PR adds support for some missing inline elements:
- emphasis/strong
- highlighted
- insert/delete
- super/subscript

I've left out inline code since Erika is already working on that in #7

### CODE REVIEW

File: test/cases_unimplemented/skipped_emphasis.test [hunk 1]
```diff
@@ -0,0 +1,35 @@
+```
```
Comment (lpil) [inline test/cases_unimplemented/skipped_emphasis.test:1]: Please call this file emphasis, as before 🙏

Comment (giacomocavalieri) [reply test/cases_unimplemented/skipped_emphasis.test:1]: Sure! I'm hoping I'll be able to move all tests to the implemented folder and I won't need this when the PR is ready

Comment (lpil) [reply test/cases_unimplemented/skipped_emphasis.test:1]: Amazing!!

**Comment by lpil** [comment]
Closing due to inactivity. Feel free to reopen. Thank you

**Pull request closed without merging**